src/main/res/values/strings.xml

```diff
@@ -45,23 +45,23 @@
   <string name="downloading_format">Downloading %s</string>
   <string name="edit_repository">Edit repository</string>
   <string name="empty_search_summary">No matching applications found</string>
-  <string name="error_downloading_format">Error downloading %s</string>
-  <string name="error_syncing_format">Error syncing %s</string>
-  <string name="error_validating_format">Error validating %s</string>
+  <string name="error_downloading_format">Could not download \"%s\"</string>
+  <string name="error_syncing_format">Could not sync \"%s\</string>
+  <string name="error_validating_format">Could not validate \"%s\</string>
   <string name="fingerprint">Fingerprint</string>
-  <string name="finished_downloading_format">Finished downloading %s</string>
+  <string name="finished_downloading_format">%s downloaded</string>
   <string name="http_error_description">Invalid server response.</string>
   <string name="http_proxy">HTTP proxy</string>
-  <string name="ignore_all_updates">Ignore all updates</string>
-  <string name="ignore_this_update">Ignore this update</string>
+  <string name="ignore_all_updates">Ignore all new versions</string>
+  <string name="ignore_this_update">Ignore this new version</string>
```

Comment:
Both texts were used in original F-Droid, I don't want to change them.

Reply:
To get the translations? Nostalgia? I am trying to avoid the Debian update/upgrade distinction used in error.

Reply:
What's the problem in translations? I don't see anything wrong and confusing in word "update", it perfectly suits this place.